src/medley/core.cljc

```diff
@@ -502,3 +502,9 @@
   []
   #?(:clj  (java.util.UUID/randomUUID)
      :cljs (cljs.core/random-uuid)))
+
+(defn regexp?
+  "Returns true if the value is a Regexp."
```

Comment:
Change "Regexp" to "regular expression", since the class in Java is called `Pattern`, not `Regexp`.